src/redux/preview.js

```diff
@@ -754,7 +754,7 @@ module.exports.updateProject = (id, jsonData, username, token) => (dispatch => {
             dispatch(module.exports.setError('No project info'));
             return;
         }
-        if (res.statusCode === 500) { // InternalServer
+        if (res.statusCode >= 400) { // InternalServer
```

Comment:
We should keep everything else up to date with this check. 500 was literally an Internal Server Error, so let's update this comment and the handler below to keep up to date with the new error codes being checked.  "API Error response" as the error that gets set, maybe?

Reply:
You're right, I was too hasty!